Code/Editor/EditorFramework/Actions/Implementation/AssetActions.cpp

```diff
@@ -124,7 +124,7 @@ void ezAssetAction::Execute(const ezVariant& value)
 
     case ezAssetAction::ButtonType::TransformAllAssets:
     {
-      ezAssetCurator::GetSingleton()->TransformAllAssets();
+      ezAssetCurator::GetSingleton()->TransformAllAssets(ezTransformFlags::TriggeredManually);
```

Comment:
This would also transform scenes etc. when clicking "transform all" in the asset browser, do we want that ?

Reply:
Probably not, I'll remove it for now.